### PR TITLE
Merge r1.5.0 bugfixes to main

### PR DESCRIPTION
**Pull request by ericharper**

### CODE REVIEW

File: nemo/collections/nlp/models/nlp_model.py [hunk 1]
```diff
@@ -249,7 +249,9 @@ def load_from_checkpoint(
                 config_kwargs.pop('trainer')
             checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY].update(config_kwargs)
 
-            model = cls._load_model_state(checkpoint, strict=strict, **kwargs)
+            model = cls._load_model_state(
+                checkpoint, strict=strict, cfg=checkpoint[cls.CHECKPOINT_HYPER_PARAMS_KEY], **kwargs
```
Comment (titu1994) [inline nemo/collections/nlp/models/nlp_model.py:253]: This PR needs to be merged first - https://github.com/NVIDIA/NeMo/pull/3172

This line is using a deprecated thing which breaks PTL for a notebook.

**Review by titu1994 — COMMENTED** [review]
A PR needs to be merged prior to this one

File: nemo/collections/nlp/models/nlp_model.py (hunk 1 above)
Comment (ericharper) [reply nemo/collections/nlp/models/nlp_model.py]: Okay I added @fayejf 's change.

**Review by titu1994 — APPROVED** [review]
LGTM !

**Pull request merged**